recipes/m17-cxx-demod/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "m17-cxx-demod" %}
+{% set version = "2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mobilinkd/m17-cxx-demod/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5dac28a7aa0c252e06b1ef1723acf0ddb6630a2a8ede07f15809cd2faef0b367
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.9
+    - make  # [not win]
+    - ninja  # [win]
+    - pkg-config  # [not win]
```

```suggestion
    - ninja
    - pkg-config
```
Switch to using `ninja` and `pkg-config` everywhere and the builds should at least proceed further.

---

Accepted.